### PR TITLE
Update to model version v0.0.135

### PR DESCRIPTION
**Pull request by zgalor**

### CODE REVIEW

**Comment by zgalor** [comment]
@tzvatot PTAL

File: servicelogs/v1/log_entry_type.go [hunk 1]
```diff
@@ -269,6 +270,29 @@ func (o *LogEntry) GetTimestamp() (value time.Time, ok bool) {
...
+	if o != nil && o.bitmap_&1024 != 0 {
+		return o.username
+	}
+	return ""
+}
+
+// GetUsername returns the value of the 'username' attribute and
+// a flag indicating if the attribute has a value.
+//
+// The username that triggered the event (if available).
```
Comment (tzvatot) [inline servicelogs/v1/log_entry_type.go:287]: NP: redundant comment

Comment (zgalor) [reply servicelogs/v1/log_entry_type.go:287]: This is generated

**Comment by zgalor** [comment]
closed and replaced by https://github.com/openshift-online/ocm-sdk-go/pull/408

**Pull request closed without merging**